doc/changes/DM-27811.removal.md

```diff
@@ -0,0 +1 @@
+Remove deprecated filterLabel exposure component access.
```

Comment:
I would argue for merging these two commits, as they're not self-contained -- if 2e0ca30 were removed, this log would be incorrect; if 4877849 were removed, the cleanup would be incomplete.